PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of products.
+
+- *Variants*   
+  Click the drop-down list to select the display settings for variants. The following settings are available:
+  - **All products**: All products and their variants are displayed in the product list.   
+  - **Hide variants**: No variants are displayed in the product list.
+  For detailed information, see [Select a variant view](../Operation/04_ManageView.md#select-a-variant-view).
+
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all products in the list are selected.
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)   
+  Click this button to edit the selected product(s). This button is only displayed, when the checkbox of at least one product is selected. Alternatively, you can click directly a row in the list to edit a product.   
+  For detailed information, see [Edit a product](../Operation/01_ManageProducts.md#edit-a-single-product).
+
+- [COPY PRODUCT]   
+  Click this button to copy the selected product. This button is only displayed, when the checkbox of a single product is selected.
+  For detailed information, see [Copy a product](../Operation/01_ManageProducts.md#copy-a-product).
+
+- [ARCHIVE PRODUCT] / [ARCHIVE PRODUCTS]   
+  Click this button to archive the selected product(s). This button is only displayed, when the checkbox of at least one product is selected.
+  For detailed information, see [Archive a product](../Operation/03_MoveProducts.md#archive-a-product).
+
+- [MOVE TO RECYCLE BIN]   
+  Click this button to move the selected product(s) to the recycle bin. This button is only displayed, when the checkbox of at least one product is selected.
+  For detailed information, see [Delete a product](../Operation/03_MoveProducts.md#delete-a-product).
+
+- [START MASS EDITING]   
+  Click this button to edit the selected products. This button is only displayed, when the checkbox of at least one product is selected. The mass editing is only recommended if you select more that one product.
+  For detailed information, see [Mass editing](../Operation/01_ManageProducts.md#mass-editing).
+
+Depending on the selection in the *Variants* drop-down list, the product list displays all products or just the master products without their variants. All fields are read-only. Depending on the settings, the displayed columns may vary.
```

Comment:
Woanders als "all products including their variants and master or single products" (ich gehe davon aus , dass master products hat variants, single product hat noch keine Variant?). Relevant, hier es auch so zu formulieren?